features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -131,6 +131,12 @@
         {% endif %}
     </ServiceProviders>
 
+    <Connectors>
+        <ChallengeQuestions>
+            <Enabled>{{connector.challenge_questions.enable}}</Enabled>
```

Comment:
```suggestion
            <Enabled>{{connectors.challenge_questions.enable}}</Enabled>
```

Reply:
IMO better to keep this as connector instead of connectors